content/ecosystem/data.toml

```diff
@@ -155,6 +155,11 @@ name = "bvh"
 source = "crates"
 categories = ["math"]
 
+[[items]]
+name = "bytemuck"
+source = "crates"
+categories = ["3drendering"]
```

Comment:
Didn't know how to categorize this

Reply:
Most of the other 'helper' libraries seem to be in the Tools category - which does make me think that should probably be renamed, tbh ('Tools and Utilities', perhaps?)

Reply:
I was a bit torn as `encase` is in "shaders" which fills the exact same role as bytemuck in most renderers, but I did look at tools. Ultimately your call :)

Reply:
Very happy with this solution 😊